teuthology/templates/email-sleep-before-teardown.jinja2

```diff
@@ -0,0 +1,9 @@
+Teuthology has sleep {{ sleep_time }}
```

Comment:
Teuthology is sleeping for {{ sleep_time }} seconds

Reply:
What about:
```Teuthology job {{ job_name }} has fallen asleep at {{ sleep_date }} for {{ sleep_time }}```